Dockerfile

```diff
@@ -4,14 +4,16 @@ RUN mkdir app
 
 WORKDIR app
 
-COPY app/package.json .
+COPY package*.json ./
```

Comment:
We have only two files falling under `package*.json` that we want in a container:
* package.json
* package-lock.json
I would rather add two entries than use regex and add any .json that's starting with package.

Reply:
True, but i'm too lazy to change it, sorry maybe later 😛